### PR TITLE
[hail] release 0.2.13

### PR DESCRIPTION
**Pull request by danking**

### CODE REVIEW

File: hail/python/hail/docs/change_log.md [hunk 1]
```diff
@@ -1,5 +1,67 @@
...
+- (hail#5692) When `keyed` is `True`, `hl.maximal_independent_set` now does not
+  produce duplicates.
+- (hail#5725) Docs now consistently refer to `hl.agg` not `agg`.
+- (hail#5730)(hail#5782) Taught `import_bgen` to optimize its `variants` argument.
+
+### Experimental
+
+- (hail#5732) `hl.agg.approx_quantiles` computes an approximation of the
+  quantiles of an aggregable.
+- (hail#5693)(hail#5396) `Table.multi_way_zip_join` now correctly handles keys
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: method starts with an underscore

File: hail/python/hail/docs/change_log.md [hunk 2]
```diff
@@ -1,5 +1,67 @@
...
+
+- (hail#5885) Fix `Table.to_spark` in the presence of fields of tuples.
+- (hail#5882)(hail#5886) Fix `BlockMatrix` conversion methods to correctly
+  handle filtered entries.
+- (hail#5884)(hail#4874) Fix longstanding crash when reading Hail data files
+  under certain conditions.
+- (hail#5855) Fix `hl.mendel_errors` incorrectly reporting children counts in
+  the presence of entry filtering.
+- (hail#5830)(hail#5835) Fix Nirvana support
+- (hail#5853)(hail#5831) Fix `mt.entries()` error when the matrix table has no
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: This bug didn't exist in the last release.

The entries() 4x performance improvement should be here though!

File: hail/python/hail/docs/change_log.md [hunk 3]
```diff
@@ -1,5 +1,67 @@
...
+- (hail#5773) Fix `hl.sample_qc` to use correct number of total rows when
+  calculating call rate.
+- (hail#5763)(hail#5764) Fix `hl.agg.array_agg` to work inside
+  `mt.annotate_rows` and similar functions.
+- (hail#5770) Hail now uses the correct unicode string encoding which resolves a
+  number of issues when a Table or MatrixTable has a key field containing
+  unicode characters.
+- (hail#5692) When `keyed` is `True`, `hl.maximal_independent_set` now does not
+  produce duplicates.
+- (hail#5725) Docs now consistently refer to `hl.agg` not `agg`.
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: I haven't been including docs stuff here, but that's fine.

File: hail/python/hail/docs/change_log.md [hunk 4]
```diff
@@ -1,5 +1,67 @@
...
+- (hail#5730)(hail#5782) Taught `import_bgen` to optimize its `variants` argument.
+
+### Experimental
+
+- (hail#5732) `hl.agg.approx_quantiles` computes an approximation of the
+  quantiles of an aggregable.
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: aggregable is a verboten word. Expression?

File: hail/python/hail/docs/change_log.md [hunk 5]
```diff
@@ -1,5 +1,67 @@
...
+- (hail#5724) `hl.rand_norm2d` samples from a two dimensional random normal.
+
+### Bug fixes
+
+- (hail#5885) Fix `Table.to_spark` in the presence of fields of tuples.
+- (hail#5882)(hail#5886) Fix `BlockMatrix` conversion methods to correctly
+  handle filtered entries.
+- (hail#5884)(hail#4874) Fix longstanding crash when reading Hail data files
+  under certain conditions.
+- (hail#5855) Fix `hl.mendel_errors` incorrectly reporting children counts in
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: add hail#5786 here?

File: hail/python/hail/docs/change_log.md [hunk 6]
```diff
@@ -1,5 +1,67 @@
...
+- (hail#5828) Remove dependency on htsjdk for VCF INFO parsing, enabling
+  faster import of some VCFs.
+- (hail#5860) Improve performance of some column annotation pipelines.
+- (hail#5858) Add `unify` option to `Table.union` which adds special behavior
+  when duplicate rows are present.
+- (hail#5756) Hail now uses Spark 2.4.x by default.
+- (hail#5677) `MatrixTable` now also supports `show`.
+- (hail#5790) Add `array.head()` which returns the first element of the array,
+  or missing if the array is empty.
+- (hail#5690) Add `_localize` flag to `locus_windows` to prevent serialization
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: I haven't been documenting underscored features. May we should promote the `localize` stuff to public?

File: hail/python/hail/docs/change_log.md [hunk 7]
```diff
@@ -1,5 +1,67 @@
...
+### New features
+
+- (hail#5828) Remove dependency on htsjdk for VCF INFO parsing, enabling
+  faster import of some VCFs.
+- (hail#5860) Improve performance of some column annotation pipelines.
+- (hail#5858) Add `unify` option to `Table.union` which adds special behavior
+  when duplicate rows are present.
+- (hail#5756) Hail now uses Spark 2.4.x by default.
+- (hail#5677) `MatrixTable` now also supports `show`.
+- (hail#5790) Add `array.head()` which returns the first element of the array,
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: array.index should be here too? #5793

File: hail/python/hail/docs/change_log.md [hunk 8]
```diff
@@ -1,5 +1,67 @@
...
+Hail is now using Spark 2.4.x by default. If you build hail from source, you
+will need to acquire this version of Spark and update your build invocations
+accordingly.
+
+### New features
+
+- (hail#5828) Remove dependency on htsjdk for VCF INFO parsing, enabling
+  faster import of some VCFs.
+- (hail#5860) Improve performance of some column annotation pipelines.
+- (hail#5858) Add `unify` option to `Table.union` which adds special behavior
```
Comment (tpoterba) [inline hail/python/hail/docs/change_log.md]: bad description - unify can unify tables with different fields or different field orders

File: hail/python/hail/docs/change_log.md (hunk 5 above)
Comment (danking) [reply hail/python/hail/docs/change_log.md]: ah, I'm looking at the actual git log which in this case did not include the updated description. 👎 GitHub

Comment (tpoterba) [reply hail/python/hail/docs/change_log.md]: ahh

File: hail/python/hail/docs/change_log.md (hunk 6 above)
Comment (danking) [reply hail/python/hail/docs/change_log.md]: probs, I changed it to a bland statement about improved ld_matrix performance

**Comment by danking** [comment]
@tpoterba back to you

**Pull request merged**